### DISCUSSION TITLE
1.0.9 Nuget Release

### DISCUSSION THREAD

**Discussion by MattB-msft** [microsoft/PowerPlatform-DataverseServiceClient#298] [General]
Fixed an issue with Custom User Agent headers incorrectly causing a format error when .net core is used. Proper User-Agent format must be used to send requests including the user agent. 
Fixed an issue with an Empty OptionSetValue collection causing an ArgumentOutOfRangeException. Empty OptionSetValue collections will now be mapped to Null. Fixes #292 
Fixed an issue where EntityReference based operations do not propagate RowVersion when using WebAPI protocol.  Fixes #296 

## What's Changed
* Resync from Main Branch - Fixes from internal and git. by @MattB-msft in https://github.com/microsoft/PowerPlatform-DataverseServiceClient/pull/297

**Full Changelog**: https://github.com/microsoft/PowerPlatform-DataverseServiceClient/compare/1.0.4...1.0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/PowerPlatform-DataverseServiceClient/releases/tag/1.0.9'>1.0.9 Nuget Release</a>.</em>

**Comment by johandynamics** [comment]
Hi,

We are currently using 1.0.23 of Microsoft.PowerPlatform.Dataverse.Client and getting some disconnect errors.
But I can't find that 1.0.23 has been released, should we go back to 1.0.9 for now or is the documentation just not updated for 1.0.23 yet?

The Date published was 10/10 2022 for 1.0.23.

**Reply by MattB-msft** [reply]
Could you open an issue on what you're seeing in the issues areas please?
